packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -126,7 +129,7 @@ export async function processOutboundDocumentQueryResps({
     // when doing the doc retrieval
     await storeInitDocRefInFHIR(docRefsWithMetriportId, cxId, patientId, log);
 
-    const respWithDRUrl: OutboundDocumentQueryResp[] = [];
+    const resultsWithMetriportIdAndDrUrl: OutboundDocumentQueryResp[] = [];
```

Comment:
Just to make sure, let's keep `response` to represent what we get from external GWs and `result` to represent the outcome of the business operation. E.g.:
- PD
  - response: each response from each external GW
  - result: the actual links
- DQ
  - response: each response from each external GW
  - result: the actual list of docs to be downloaded (to be sent to DR)
- ...

Not sure if in this case were referring to the response or result, but it would be good to keep consistency.

Reply:
In this case, according to the `result` vs `response` logic you laid out, since this is the list of documents about to be sent to DR, the naming is fine. 

But tbh this distinction you laid out isnt really followed and I have never heard us say it before. The overall function takes the resp immediately from the gateway but calls it resullts. Same goes for xcpd [and](https://github.com/metriport/metriport/blob/2bdc346ed0cb77e18d464bd92ad30509563c0a7d/packages/core/src/external/carequality/ihe-gateway/outbound-result-poller-direct.ts#L16) dr in their respective `processOutboundXyz` functions

Reply:
At least for XCPD I'm sure the above is correct; for DQ and "..." I just applied the same logic, but maybe its not consistent for those yet.